pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -886,6 +894,28 @@ void GraphicsSettingsWidget::onShadeBoostChanged()
 	m_ui.shadeBoostSaturation->setEnabled(enabled);
 }
 
+void GraphicsSettingsWidget::onMessagesPosChanged()
+{
+	const bool enabled = m_ui.osdMessagesPos->currentIndex() != (m_dialog->isPerGameSettings() ? 1 : 0);
```

Comment:
I don't follow the logic here. You're checking the if the current position index is not equal to if you're modifying per game settings?

Reply:
The logic here is that for the per-game settings, the `Global [Default]` is index 0 instead of `None`. Consequently, if it's just `!= 0`, then the global settings work, but the per-game ones do not.

Reply:
Ahh I see now.